database/logging/file.go

```
@@ -0,0 +1,72 @@
+package logging
+
+/**
+ * This database type is just for,
+ * - debugging without a influxconn
+ * - example for other developers for new databases
+ */
+import (
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"github.com/FreifunkBremen/yanic/database"
+	"github.com/FreifunkBremen/yanic/runtime"
+)
+
+type Connection struct {
+	database.Connection
+	config Config
+	file   *os.File
+}
+
+type Config map[string]interface{}
+
+func (c Config) Enable() bool {
+	return c["enable"].(bool)
+}
+func (c Config) Path() string {
+	return c["path"].(string)
+}
+
+func init() {
+	database.AddConnect("logging", Connect)
+}
+
+func Connect(configuration interface{}) database.Connection {
```

Don't ignore errors - return them.